### PR TITLE
Shake

### PR DESCRIPTION
**Pull request by gables**
Added a little more detail to the main title menu. Still needs lots of work, feel free to add, change and tinker further.

### CODE REVIEW

File: checkers.kv [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<TitleMenu>:
+    BoxLayout:
+        orientation: 'vertical'
+        Label:
+            text: 'Checkers'
+        Button:
```
Comment (DanielBeer) [inline checkers.kv]: As discussed on Discord, as these settings pertain to the instance of the game, they should only be accessible through "Play" being clicked

File: checkers.kv [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+            text: 'Settings'
+            on_release: root.manager.current = 'settings'
+        Button:
+            text: 'Play'
```
Comment (DanielBeer) [inline checkers.kv]: Now that we have the screen, might as well link this to settings as mentioned above

File: checkers.kv [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<TitleMenu>:
+    BoxLayout:
+        orientation: 'vertical'
```
Comment (DanielBeer) [inline checkers.kv]: As we're building for desktop shouldn't these be horizontal?

File: checkers.kv [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+            max: 10
+            value: 3
+            step: 1
+            orientation: 'horizontal'
+        Label:
+            text: str(round(slider.value))
+
+<SettingsMenu>:
+    BoxLayout:
+        orientation: 'vertical'
```
Comment (DanielBeer) [inline checkers.kv]: Horizontal?

File: checkers.kv [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        Label:
+            text: 'Settings'
+        PlayerSetting:
+        DifficultySetting:
```
Comment (DanielBeer) [inline checkers.kv]: Difficulty setting should only be visible if the player setting is 2 player

File: checkers.kv (hunk 1 above)
Comment (rgables) [reply checkers.kv]: I don't totally follow.  The settings affect how the game is set up.  When the user hits play, don't we want the game to start in the correct mode? Player v cpu / player v player

File: checkers.kv (hunk 3 above)
Comment (rgables) [reply checkers.kv]: In its current state horizontal menu buttons are a little awkward.  I'm sure all this stuff will get moved around as the app takes shape.

File: checkers.kv (hunk 5 above)
Comment (rgables) [reply checkers.kv]: That's good feedback, let me see what i can do

**Review by DanielBeer — APPROVED** [review]

**Pull request merged**